packages/sanity/src/core/studio/components/navbar/search/components/searchResultItem/SearchResultItemPreview.tsx

```diff
@@ -0,0 +1,86 @@
+import type {SanityDocument} from '@sanity/client'
+import type {SchemaType} from '@sanity/types'
+import {Box, Inline, Label} from '@sanity/ui'
+import React, {useMemo} from 'react'
+import {useMemoObservable} from 'react-rx'
+import styled from 'styled-components'
+import {DocumentPreviewPresence} from '../../../../../../presence'
+import {
+  DocumentPreviewStore,
+  getPreviewStateObservable,
+  getPreviewValueWithFallback,
+  SanityDefaultPreview,
+} from '../../../../../../preview'
+import type {DocumentPresence} from '../../../../../../store'
+
+interface SearchResultItemPreviewProps {
+  documentId: string
+  documentPreviewStore: DocumentPreviewStore
+  presence?: DocumentPresence[]
+  schemaType: SchemaType
+}
+
+/**
+ * Brute force all nested boxes on iOS to use `background-attachment: scroll`, as
```

Comment:
Does sanity/ui have a task for this?

Reply:
There's no task as this isn't so much an issue with `@sanity/ui`, but rather a temporary workaround to allow Skeleton components to render correctly within virtual lists, which is definitely an edge case. There's probably a better way to handle this.

This comment was also inaccurate re: `background-attachment: fixed` support on iOS and has been updated